lib/segment/src/index/field_index/numeric_index/mutable_numeric_index.rs

```diff
@@ -1,28 +1,53 @@
 use std::collections::BTreeSet;
 use std::ops::Bound;
 use std::ops::Bound::{Excluded, Unbounded};
+use std::path::PathBuf;
 use std::sync::Arc;
 
 use common::counter::hardware_counter::HardwareCounterCell;
 use common::types::PointOffsetType;
+use gridstore::config::StorageOptions;
+use gridstore::{Blob, Gridstore};
 use parking_lot::RwLock;
 use rocksdb::DB;
 
 use super::mmap_numeric_index::MmapNumericIndex;
 use super::{
     Encodable, HISTOGRAM_MAX_BUCKET_SIZE, HISTOGRAM_PRECISION, numeric_index_storage_cf_name,
 };
+use crate::common::Flusher;
 use crate::common::operation_error::{OperationError, OperationResult};
 use crate::common::rocksdb_buffered_delete_wrapper::DatabaseColumnScheduledDeleteWrapper;
 use crate::common::rocksdb_wrapper::DatabaseColumnWrapper;
 use crate::index::field_index::histogram::{Histogram, Numericable, Point};
 use crate::index::field_index::mmap_point_to_values::MmapValue;
 
-pub struct MutableNumericIndex<T: Encodable + Numericable> {
-    db_wrapper: DatabaseColumnScheduledDeleteWrapper,
+/// Default page size used in Gridstore
+pub const DEFAULT_PAGE_SIZE_BYTES: usize = 4 * 1024 * 1024; // 4 MiB = 16 * 8192 * 32
```

Comment:
I think where the number comes from could be a bit clearer 😅 
```suggestion
pub const DEFAULT_PAGE_SIZE_BYTES: usize = 16 * 8192 * 32; // 4 MiB = block_size * region_blocks * regions
```